apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -220,9 +220,17 @@ func (m *ModelManagerAPI) CreateModel(ctx context.Context, args params.ModelCrea
 			return result, errors.Trace(err)
 		}
 		creationArgs.Credential = credential.KeyFromTag(cloudCredentialTag)
+	} else {
+		// we're looking up credentials for the apiUser - what if JIMM is creating a model on behalf of another user (JIMM always
+		// specifies the credential, so we should be fine)
+		credKey, _, err := m.credentialService.SingleValidCloudCredentialsForOwner(ctx, coreuser.NameFromTag(m.apiUser), cloudTag.Id())
+		if err != nil {
+			return result, errors.Trace(err)
```

Comment:
We can probably forgoe some error handling here because we know things will be checked in other places like user and cloud. But if we are doing this properly they need to be asserted and errors handled.